app/components/ActivityVolonteersList.jsx

```diff
@@ -37,7 +37,7 @@ var ActivityVolonteersList = React.createClass ({
           return {
             display_name: option.text,
             user_id: option.payload.id,
-            thumb_picture_url: option.payload.thumb_picture_url
+            thumb_picture_url: option.payload.profile_picture_url
```

Comment:
Bo nie wyświetlało się w szukajce zdjęcia i jak sobie wyprintowałem całą zawartość option.payload to okazało się że thumb_picture_url jest puste a w profile_picture_url są zdjęcia